make/cluster.sh

```diff
@@ -36,10 +36,12 @@ Usage:
     ${bold}$0 [--mode kind] [--name NAME] [--show-image] [--help]${end}
 
 Flags:
+  # TODO: do we need this flag? kind is the only mode possible at the moment.
```

Comment:
suggestion: kill it with fire now IMO if you want to, I'd `/lgtm` that in this PR!

Reply:
ideally I guess I'd like to remove all of them as I think this bit https://github.com/cert-manager/cert-manager/blob/master/make/cluster.sh#L58-L93 is a little too complicated and isn't really serving any purpose